leo/commands/package/publish.rs

```diff
@@ -15,28 +15,22 @@
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
 use crate::{
-    cli::*,
-    cli_types::*,
-    commands::{BuildCommand, LoginCommand},
-    config::{read_token, PACKAGE_MANAGER_URL},
-    errors::{
-        commands::PublishError::{ConnectionUnavalaible, PackageNotPublished},
-        CLIError,
-        PublishError::{MissingPackageDescription, MissingPackageLicense, MissingPackageRemote},
-    },
-};
-use leo_package::{
-    outputs::OutputsDirectory,
-    root::{Manifest, ZipFile},
+    commands::Command,
+    context::{Context, PACKAGE_MANAGER_URL},
 };
 
-use clap::ArgMatches;
+use anyhow::{anyhow, Result};
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates